src/main/java/org/folio/rest/impl/BaseApi.java

```diff
@@ -74,7 +75,11 @@ public void addProcessingError(Error error) {
   }
 
   protected int handleProcessingError(Throwable throwable) {
-    final Throwable cause = throwable.getCause();
+    Throwable cause = throwable.getCause();
+    if(cause == null)
+    {
+      cause = throwable.initCause(new CompletionException(throwable));
+    }
```

Comment:
Why this code introduced, and it should be formetted to use the same style as in other places

Reply:
when we return HttpException directly ,it have only error code and message. it does not have cause so cause is null due to that it is not throwing original exception rather it return bad request. 
so i am initializing cause when cause is null .